sklearn/feature_selection/_univariate_selection.py

```diff
@@ -317,8 +317,9 @@ def f_regression(X, y, *, center=True, force_finite=True):
     This is done in 2 steps:
 
     1. The cross correlation between each regressor and the target is computed,
-       that is, ((X[:, i] - mean(X[:, i])) * (y - mean_y)) / (std(X[:, i]) *
-       std(y)) using r_regression function.
+       that is, mean((X[:, i] - mean(X[:, i])) * (y - mean(y))) / (std(X[:, i])
```

Comment:
Normally, I see this using expectation: 

```
       that is, E[(X[:, i] - mean(X[:, i])) * (y - mean(y))] / (std(X[:, i])
```

Do you think this formulation is clearer than using nested means?

Reply:
Honestly my gut reaction is to prefer the nested mean, just because it stays consistent with the in-line notation, and I worry the expectation may create ambiguity (e.g. why did the docs _not_ use mean?). I feel like the nested mean is the better at-a-glance documentation